cpp/core/jni/JniWrapper.cc

```diff
@@ -959,19 +959,17 @@ JNIEXPORT void JNICALL Java_io_glutenproject_vectorized_OnHeapJniByteInputStream
 JNIEXPORT jlong JNICALL Java_io_glutenproject_vectorized_ShuffleReaderJniWrapper_make( // NOLINT
     JNIEnv* env,
     jobject,
-    jobject jniIn,
     jlong cSchema,
     jlong allocId,
     jstring compressionType,
     jstring compressionBackend) {
   JNI_METHOD_START
+  ReaderOptions options = ReaderOptions::defaults();
```

Comment:
hmm, why need to move from L974 to L967?

Reply:
fixed